src/modules/layout/layout-blank.component.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { NO_ERRORS_SCHEMA } from '@angular/core';
```

Comment:
I thought it was for templates at first, but then realized the error was something else. I'll remove it.